src/sentry/runner/commands/devservices.py

```diff
@@ -13,6 +13,29 @@
     os.environ["DOCKER_HOST"] = "unix://" + RAW_SOCKET_HACK_PATH
 
 
+def start_docker_client():
+    import subprocess
+    import sys
+
+    import docker
+
+    if sys.platform == "darwin":
+        subprocess.run(["open", "--background", "-a", "Docker"])
+    else:
+        raise click.ClickException("Make sure docker in running.")
+
+    maxWait = 30
+    start = time.time()
+    while time.time() - start < maxWait:
```

Comment:
generally you'll want to use `time.monotonic()` so it is not subject to clock drift (or `perf_counter` if you ever expect this to run on windows and need more accuracy (I don't))